prometheus/rootfs/bin/boot.sh

```diff
@@ -22,18 +22,23 @@ done
 
 set +e
 
-OPTS=""
+until [ -n "${DEIS_MONITOR_ALERT_SERVICE_HOST+1}" ]; do
```

Comment:
If this environment variable isnt set initially, will it somehow be be set once the service starts? I assumed that was not the case, but that would sweet if it was.